rdr_service/workflow_management/nph/sms_workflows.py

```diff
@@ -146,6 +159,16 @@ def job_ingestion(self):
         Main method for ingestion jobs.
         """
 
+        # Ensure file is CSV
+        if not str(self.file_path).endswith(".csv"):
+            file_end = self.file_path.split(".")[-1]
+            nph_slack = get_nph_slack_message_handler()
+            nph_slack.send_message_to_webhook(
+                message_data=f"Error ingesting nph file of type {file_end}. "
+                             "File does not conform to csv format."
```

Comment:
Can you add the `file_path` to the message as well? Something like "File {file_path} does not conform to csv format."